create-reporter/__snapshots__/index.test.js.snap

```diff
@@ -255,7 +255,6 @@ exports[`reports listen 2`] = `
         Subprotocol:    [1m0.0.0[22m
         Supports:       [1m0.x[22m
         Control listen: [1mhttp://127.0.0.1:31337/[22m
-        Control mask:   [1mundefined[22m
```

Comment:
"undefined" values are missing from logs, because json formatted string is passed to HumanFormatter.
https://github.com/pinojs/pino/issues/618